website/src/roadmap/index.md

```diff
@@ -70,6 +70,13 @@ streaming pipelines using only SQL, but also to allow Beam Java developers
 to use SQL in components of their pipeline for added efficiency. See the 
 [Beam SQL Roadmap]({{site.baseurl}}/roadmap/sql/)
 
+## Portable schemas
+
+Schemas describe types in terms of fields and relieve the user from having to write custom
```

Comment:
Users mostly already don't have to write coders. I think schema "allow SDKs and runners to understand the structure of user data, and to share that understanding, for example enabling seemless compatibility between rows in Python and Java, and unlocking relational optimization possibilities"